features/org.wso2.carbon.identity.application.authenticator.samlsso.server.feature/resources/samlsso_federate.html

```diff
@@ -97,14 +103,11 @@
                     <i class="fa fa-circle-o-notch fa-spin loader-redirect"></i>
                     Please wait while we take you to the $url.
                 </div>
-                <p class="padding-top-2 padding-bottom-1 text-align-center">Click continue if you have been waiting for
+                <p class="padding-bottom-1 text-align-center">Click <span class="primary-color-btn button" onclick="javascript:document.getElementById('outbound-auth-samlsso-form').submit()">here</span> if you have been waiting for
```

Comment:
Shall we use an anchor tag here in stead wrapping up with a span tag?

Reply:
Fixed